scripts/fvt/db_test.sh

```diff
@@ -261,63 +261,3 @@ killserver $pid || ErrorExit "Failed to stop CA"
 
 checkIdentity "a" $SERVERLOG # Check to see that a new identity properly got registered
 checkIdentity "c" $SERVERLOG # Check to see that a new identity properly got registered
-
-echo "############################ PostgresSQL Test with Client ############################"
-
-kill -INT `head -1 /usr/local/pgsql/data/postmaster.pid` # Shutdown postgres server
-pollPostgres "" "" "" stop 2>&1 # Wait for postgres to stop
-
-# Start fabric-ca server connecting to postgres, this will fail
-SERVERLOG="$FABRIC_CA_SERVER_HOME/serverlog.test1c.txt"
-$SCRIPTDIR/fabric-ca_setup.sh -S -X -g $PGSQLSERVERCONFIG2 | tee $SERVERLOG 2>&1 &
-pollLogForMsg "Listening on https*://0.0.0.0:$CA_DEFAULT_PORT" $SERVERLOG || ErrorExit "Failed to log CA startup message"
-
-# Enroll with a server that does not have a DB initialized, should expect to get back error
-enroll a b 2>&1 | grep "Failed to connect to Postgres database"
-if [ $? != 0 ]; then
-    ErrorMsg "Enroll request should have failed due to uninitialized postgres database"
-fi
-
-# Start postgres server
-su postgres -c 'postgres -D /usr/local/pgsql/data' &
-pollPostgres # Wait for postgres to start
-sleep 5 # Postgres port is available but sometimes get back 'pq: the database system is starting up' error. Putting in sleep to allow for start up to complete
-
-# Enroll again, this time the server should try to reinitialize the DB before processing enroll request and this should succeed
-enroll a b 2>&1 | grep "Stored client certificate"
-if [ $? != 0 ]; then
-    ErrorMsg "Enroll request should have passed"
-fi
-
-$SCRIPTDIR/fabric-ca_setup.sh -K
-
-echo "############################ MySQL Test with Client ############################"
-
-/etc/init.d/mysql stop >/dev/null 2>&1
-pollMySql "" "" "" stop # Wait for MySQL to stop
-
-# Start fabric-ca server connecting to MySQL, this will fail
-SERVERLOG="$FABRIC_CA_SERVER_HOME/serverlog.test2c.txt"
-$SCRIPTDIR/fabric-ca_setup.sh -S -X -g $MYSQLSERVERCONFIG2 | tee $SERVERLOG 2>&1 &
-pollLogForMsg "Listening on https*://0.0.0.0:$CA_DEFAULT_PORT" $SERVERLOG || ErrorExit "Failed to log CA startup message"
-
-# Enroll with a server that does not have a DB initialized, should expect to get back error
-enroll a b 2>&1 | grep "Failed to connect to MySQL database"
-if [ $? != 0 ]; then
-    ErrorMsg "Enroll request should have failed due to uninitialized mysql database"
-fi
-
-# Start mysql server
-/usr/bin/mysqld_safe --sql-mode=STRICT_TRANS_TABLES &
-pollMySql # Wait for MySQL to start
-
-# Enroll again, this time the server should try to reinitialize the DB before processing enroll request and this should succeed
-enroll a b 2>&1 | grep "Stored client certificate"
-if [ $? != 0 ]; then
-    ErrorMsg "Enroll request should have passed"
-fi
-
-$SCRIPTDIR/fabric-ca_setup.sh -R -x $FABRIC_CA_SERVER_HOME
-
-CleanUp $RC
-exit $RC
```

Comment:
Not sure what to do about these two tests. They stop the database service, create a new CA, start the DB, then restart the server to verify DB initialization. Not possible for us to start and stop service containers